data/STRchive-loci.json

```diff
@@ -1129,7 +1129,7 @@
   "mechanism": "GoF",
   "mechanism_detail": "Aberrant splicing, RAN translation [@pmid:22140091; @pmid:38467784].",
   "source": [],
-  "details": "Detailed overview of disease locus through 2024 by Rimoldi et al [@pmid:39643839]. ≤30 uninterrupted CCTG repeats or 11-26 CCTG repeats with GCTC/TCTG interruptions are considered benign; 27-29 repeats with interruptions have currently unknown significance, ~30-~54 repeats are considered premutations, ~55-74 repeats are premutations with possible reduced penetrance, and >74 repeat alleles are considered pathogenic [@genereviews:NBK1466]. Penetrance is age-dependent and approaches 100%. Locus structure is (TG)n(TCTG)n(CCTG)n. CCTG expansion causes DM2 but the other repeat units are also variable. Interruptions include GCTG/TCTG/GGCT [@pmid:35245110].",
+  "details": "Detailed overview of disease locus through 2024 by Rimoldi et al [@pmid:39643839]. ≤30 uninterrupted CCTG repeats or 11-26 CCTG repeats with GCTC/TCTG interruptions are considered benign; 27-29 repeats with interruptions have currently unknown significance, ~30-~54 repeats are considered premutations, ~55-74 repeats are premutations with possible reduced penetrance, and >74 repeat alleles are considered pathogenic [@genereviews:NBK1466]. Penetrance is age-dependent and approaches 100%. Locus structure is (TG)n(TCTG)n(CCTG)n. CCTG expansion causes DM2 but the other repeat units are also variable. Interruptions include GCTG/TCTG/GGCT [@pmid:35245110; @pmid:39703464].",
```

Comment:
Can you add something about how to interpret CAGA? Are they usually just interruptions and do they affect pathogenicity? What would make an interruption become a pathogenic motif?

Reply:
Perhaps it would be better to put TCTG motif under the unknown_motif category and add some of your comment above to the details.